api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/activation/ActivationTest.kt

```diff
@@ -31,4 +31,16 @@ open class ActivationTest {
             )
         }
     }
+
+    protected fun Activation.apply(input: FloatArray): FloatArray = EagerSession.create().use { session ->
+        val tf = Ops.create(session)
+
+        val operand = this.apply(tf, tf.constant(input))
+
+        operand.asOutput().tensor().copyTo(FloatArray(input.size))
+    }
+
+    protected fun assertActivationFunction(activation: Activation, input: FloatArray, expected: FloatArray) {
```

Comment:
What do you think about `assertActivationFunction` without `actual` parameter - as I see in most (all?) cases the caller just pass an array of zeroes, which can be prepared inside `assertActivationFunction`?

Reply:
Yeah, you are absolutely right about the `actual` parameter, it's just an empty array for copying needs